### ISSUE TITLE
Provide multi online version for different Jakarta EE version?

### ISSUE THREAD

**Issue by hantsy** [eclipse-ee4j/jakartaee-tutorial#252]
I think it is better to provide multi versions for different Jakarta EE versions.

Such as 
* http://xxx/index.html like the Jakarta EE spec on Jakarta.ee, provide an index page for all maintained versions.
* http://xxx/8/index.html for Jakarta EE 8
* http://xxx/9/index.html for Jakarta EE 9 and Jakarta EE 9.1
* http://xxx/current/index.html for the latest development version

**Comment by arjantijms** [comment]
+1, I was just thinking exactly the same ;)

**Comment by keilw** [comment]
Yes indeed, it has a Draft 9 already, but either accept that http://xxx/8/index.html for Jakarta EE 8 would be this project's 8.1 release or it would have to be http://xxx/8.1/index.html instead?

**Comment by flux77** [comment]
Yes. Considering that older versions of Jakarta EE are still widely used, it makes sense to host online copies of tutorials for the older versions of Jakarta EE. Failing that, the older tutorials should be provided as a ZIP file download for offline usage.

For example, Oracle is still hosting the documentation (including tutorials) for old versions of Java EE:
* Java EE 7: https://docs.oracle.com/javaee/7/
* Java EE 6: https://docs.oracle.com/javaee/6/
* Java EE 5: https://docs.oracle.com/javaee/5/

Jakarta EE should do the same.
